### PR TITLE
Add Alignak-app in presentation

### PR DESCRIPTION
**Pull request by algorys**
I've not update the schema @ddurieux, not sure it is necessary :hammer:

### CODE REVIEW

**Comment by ddurieux** [comment]
I think miss the most important information: 
alignak-app is an application in systrayicon prevent user by screen notification when something is wrong

**Comment by mohierf** [comment]
I would write that: *alignak-app is a desktop application for Linux or Windows to alert the end-user with popup notifications when a problem happens. It also allow the user to select an host to view its current state.*

**Comment by algorys** [comment]
@mohierf it seems a good description.

File: presentation/index.html [hunk 1]
```diff
@@ -197,6 +197,43 @@
 
 </section>
 
+<section data-markdown>
+<script>
+## Alignak-app
+
+* Alignak-app is a desktop application, in the system tray, for Linux or Windows. <!-- .element: class="fragment" data-fragment-index="1" -->
+* She alert the end-user with popup notifications, when a problem happens. <!-- .element: class="fragment" data-fragment-index="2" -->
```
Comment (mohierf) [inline presentation/index.html]: **It** alert**s** :wink:

Comment (algorys) [inline presentation/index.html]: Fix

**Comment by algorys** [comment]
@ddurieux and @mohierf fixed.

**Comment by mohierf** [comment]
@ddurieux : please merge and update site :wink:

**Pull request merged**